ProjectLighthouse.Servers.API/Controllers/UserEndpoints.cs

```diff
@@ -128,18 +128,23 @@ public async Task<IActionResult> CreateUserInviteToken([FromRoute] string? usern
     /// <summary>
     /// Gets a list of online users and returns it.
     /// </summary>
+    /// <param name="page">Page number. Increments skip value by clamp value</param>
+    /// <param name="clamp">Number of entries to clamp response to</param>
     /// <returns>Array of online users ordered by login time</returns>
     /// <response code="200">List of users</response>
     [HttpGet("users/online")]
     [ProducesResponseType(typeof(ApiUser), StatusCodes.Status200OK)]
-    public async Task<IActionResult> GetOnlineUsers()
+    public async Task<IActionResult> GetOnlineUsers(int page = 0, int clamp = 50)
     {
+        if (clamp > 100) return this.BadRequest();
+
         List<ApiUser> onlineUsers = (await this.database.Users
             .Where(u => u.PermissionLevel != PermissionLevel.Banned)
             .Where(u => u.ProfileVisibility == PrivacyType.All)
             .Where(u => u.GetStatus(this.database).StatusType == StatusType.Online)
             .OrderBy(u => u.LastLogin)
-            .Take(50)
+            .Skip(page * clamp)
+            .Take(clamp)
```

Comment:
This could probably be moved to it's own query extension in the future as well as cleaned up so it can be more easily used, but that's outside the scope of this PR.